tests/asr/keyword/test_tflite_recognizer.py

```diff
@@ -0,0 +1,88 @@
+"""
```

Comment:
the convention for tests is that the module name should be the same as the module under test, with the "test_" prefix